.buildkite/scripts/steps/integration_tests_tf.sh

```diff
@@ -39,13 +39,19 @@ if [[ "${BUILDKITE_RETRY_COUNT}" -gt 0 || "${FORCE_ESS_CREATE:-false}" == "true"
 else
   # For the first run, we start the stack in the start_ess.sh step and it sets the meta-data
   echo "~~~ Receiving ESS stack metadata"
-  export ELASTICSEARCH_HOST=$(buildkite-agent meta-data get "es.host")
-  export ELASTICSEARCH_USERNAME=$(buildkite-agent meta-data get "es.username")
-  export ELASTICSEARCH_PASSWORD=$(buildkite-agent meta-data get "es.pwd")
-  export KIBANA_HOST=$(buildkite-agent meta-data get "kibana.host")
-  export KIBANA_USERNAME=$(buildkite-agent meta-data get "kibana.username")
-  export KIBANA_PASSWORD=$(buildkite-agent meta-data get "kibana.pwd")
-  export INTEGRATIONS_SERVER_HOST=$(buildkite-agent meta-data get "integrations_server.host")
+  if [[ $BUILDKITE_STEP_KEY == "integration-fips-ess" ]]; then
+    JOB_ID=$(buildkite-agent meta-data get "ess.job.fips")
+  else
+    JOB_ID=$(buildkite-agent meta-data get "ess.job")
+  fi
+  echo "Using JOB_ID: ${JOB_ID}"
+  export ELASTICSEARCH_HOST=$(buildkite-agent meta-data get "es.host" --job ${JOB_ID})
+  export ELASTICSEARCH_USERNAME=$(buildkite-agent meta-data get "es.username" --job ${JOB_ID})
+  export ELASTICSEARCH_PASSWORD=$(buildkite-agent meta-data get "es.pwd" --job ${JOB_ID})
+  export KIBANA_HOST=$(buildkite-agent meta-data get "kibana.host" --job ${JOB_ID})
+  export KIBANA_USERNAME=$(buildkite-agent meta-data get "kibana.username" --job ${JOB_ID})
+  export KIBANA_PASSWORD=$(buildkite-agent meta-data get "kibana.pwd" --job ${JOB_ID})
+  export INTEGRATIONS_SERVER_HOST=$(buildkite-agent meta-data get "integrations_server.host" --job ${JOB_ID})
```

Comment:
you need to do the same for the ps1 (windows powershell) script. Look [here](https://buildkite.com/elastic/elastic-agent/builds/24225#019819e2-60b3-4203-ac66-c3e44a978e67/140-330) and [here](https://buildkite.com/elastic/elastic-agent/builds/24225#019819e2-629e-45e0-b652-207ab09947a0/764-765). Evidently when you don't specify the job this is getting any it finds?!

please add a an echo to the ELASTICSEARCH_HOST also for in this file to be able to tell which ES each runner is using and from where 🙂

Reply:
done